### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by evagoras**
Need to CD into the "vagrant" folder to start it up

### CODE REVIEW

**Comment by JoshHighland** [comment]
good catch

File: vagrant/provisioners/install-lucee.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ sudo /bin/cp -f /vagrant/configs/lucee-server.xml /opt/lucee/lib/lucee-server/co
 sudo cp /vagrant/configs/server.xml /opt/lucee/tomcat/conf/server.xml
 
 sudo mkdir -p /opt/lucee/config/web
-sudo cp -f /vagrant/configs/web.xml /opt/lucee/tomcat/conf/web.xml
```
Comment (bdw429s) [inline vagrant/provisioners/install-lucee.sh:66]: The file `web.xml` is actually correct.  It is needed to override the web context location to the folder path created on the above line.  The issue is that the file apparently didn't get committed to our repo.  If you put this line back, I'll merge the rest of the pull.  I'll grab a correct version of the web.xml, or you can try your hand at it :)

Comment (evagoras) [inline vagrant/provisioners/install-lucee.sh:66]: Actually, just deny this request. The updated libraries don't work with each other so well. I did a fresh install of vagrant with the changes and it's running into issues. Until I have time to sort them out, just cancel the pull.
I did try to add my own web.xml file in there, but it didn't work. If you have the correct one, please do so I can test again.

Comment (bdw429s) [inline vagrant/provisioners/install-lucee.sh:66]: What kind of issues are you running into?

Comment (evagoras) [inline vagrant/provisioners/install-lucee.sh:66]: The Lucee updated patch doesn't seem to work. I get a whole stack of Tomcat dumps. The CommandBox update seems to work just fine.

**Comment by lmajano** [comment]
closed as requested

**Comment by evagoras** [comment]
Reminding you to add the `web.xml` file missing from inside the `/vagrant/configs/` folder. I can test once it's in there.

**Comment by bdw429s** [comment]
Thanks for the bump...

**Comment by bdw429s** [comment]
Ok @evagoras I've added the web.xml and updated Lucee.  Please pull a fresh copy and try again.

**Pull request closed without merging**